gunicorn_config.py

```diff
@@ -2,10 +2,6 @@
 import sys
 import traceback
 
-import newrelic.agent  # See https://bit.ly/2xBVKBH
-
-newrelic.agent.initialize()  # noqa: E402
```

Comment:
This should not be removed because Kubernetes won't report New Relic metrics anymore.

Reply:
Ok will revert